<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Dong" ignored="0" name="TestngPro" tests="3" failures="1" timestamp="22 一月 2018 07:38:42 GMT" time="0.049" errors="0">
  <testcase name="testFactorial" time="0.004" classname="com.dong.testng.test.MathTest">
    <failure type="java.lang.AssertionError" message="expected [6] but found [5]">
      <![CDATA[java.lang.AssertionError: expected [6] but found [5]
	at com.dong.testng.test.MathTest.testFactorial(MathTest.java:21)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testFactorial -->
  <testcase name="testFactorial2" time="0.0" classname="com.dong.testng.test.MathTest"/>
  <testcase name="testFactorial1" time="0.0" classname="com.dong.testng.test.MathTest">
    <skipped/>
  </testcase> <!-- testFactorial1 -->
</testsuite> <!-- TestngPro -->
